Example/PaymentSheet Example/PaymentSheet Example/EmbeddedPlaygroundViewController.swift

```diff
@@ -8,8 +8,14 @@
 import Foundation
 @_spi(EmbeddedPaymentElementPrivateBeta) @_spi(STP) @_spi(ExperimentalAllowsRemovalOfLastSavedPaymentMethodAPI) import StripePaymentSheet
 import UIKit
+import Combine
+import SwiftUI
 
 class EmbeddedPlaygroundViewController: UIViewController {
+    private var hostingController: UIHostingController<AnyView>?
+    private var cancellables = Set<AnyCancellable>()
+    private var playgroundController: PlaygroundController
```

Comment:
Seems like `playgroundController` should be a weak reference to avoid a retain cycle?

Reply:
This shouldn't happen, but I re-wrote this so it's better!